content/lang/collections/loans/loan_langs.en.php

```diff
@@ -60,7 +60,7 @@
 $LANG['ADD_NEW_INST'] = 'Add a New Institution';
 $LANG['CREATE_EXCHANGE'] = 'Create Exchange';
 $LANG['TRANS_BY_INST'] = 'Transaction Records by Institution';
-$LANG['OUTGOING_EX_SENT'] = 'Outgoing exchange; Sent ';
+$LANG['BARCODE_CATNUM'] = 'Outgoing exchange; Sent ';
```

Comment:
I think there's still a call to `$LANG['OUTGOING_EX_SENT']` in collections/loans/exchangetab.php. Make sure that you're searching for these translations globally before replacing or removing them.